admin-dev/themes/new-theme/package-lock.json

```diff
@@ -1,14275 +1,8 @@
 {
   "name": "new-theme",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Be careful I don't think the package lock i supposed to be committed in this case, and it seems you used a different node version thus it changes the whole structure. We need to find. away to prevent this via a CI maybe so that we have consistency with our package lock files.
But unless you needed to update one of the JS dependencies I think you should revert this change

Reply:
oh I wasn't intending to commit this, will revert, thanks for noticing